CHANGELOG.md

```diff
@@ -14,11 +14,22 @@
 
 🆕 New features:
 
-- Pull Request Title goes here
+- Enable GOV.UK Frontend components to be scoped the a specific container
```

Comment:
@alex-ju would you mind keeping the template around instead of overwriting it, this is so future people can copy and paste it easier.

Reply:
sure, added it back